hydroflow_datalog_core/src/lib.rs

```diff
@@ -0,0 +1,336 @@
+use std::collections::{HashMap, HashSet};
```

Comment:
I'm assuming this file is effectively the same as the old `hydroflow_datalog/src/lib.rs`

Reply:
yep!